src/Modal/index.jsx

```diff
@@ -183,6 +185,7 @@ class Modal extends React.Component {
               [styles['d-block']]: open,
               [styles.show]: open,
               [styles.fade]: !open,
+              [styles['is-ie']]: isIE11,
```

Comment:
Should there be a test for this?

Reply:
How would I test this? I would have to mimic an IE11 browser and I don't think we have the ability to do that in our tests.

Reply:
Actually, I found a way to mock globals in Jest. I'll push up a test in sec.